source/install/desktop-custom-dictionaries.rst

```diff
@@ -0,0 +1,32 @@
+Desktop App Custom Dictionaries
+===========================
+
+Mattermost Desktop App users running Windows or Linux can download dictionary definitions from a custom URL. If custom dictionaries aren't specified, default dictionary definitions are obtained automatically from Chromium's CDNs.
+such a service, dictionaries definitions will be automatically obtained from Chromium's CDNs.
```

Comment:
We can safely delete this line of text: "such a service, dictionaries definitions will be automatically obtained from Chromium's CDNs."

Reply:
oh! removed the next one but somehow I missed this one, will fix it now